Examples.md

```diff
@@ -3,3 +3,21 @@ You want to see the examples? Did you even read all the features of DreamBerd? O
 
 Please go back to the features page by following [this link](https://github.com/TodePond/DreamBerd/blob/main/README.md).<br>
 When you have finished reading the features, you can find the _real_ examples [here](https://github.com/TodePond/DreamBerd/blob/main/res/Examples.md).
+
+```DREAMBERD PUBLIC LICENSE
+Version 1, April 2023
+
+Copyright (C) 2023 Lu Wilson <l2wilson94@gmail.com>
+
+Everyone is permitted to release software, as long as they follow the terms and conditions listed below.
```

Comment:
This wording is ambiguous, please change to:
```
Everyone is not permitted to release spftware, as long as they do not follow the terms and conditions listed above.
```

And then maybe put the conditions above (aka in the parent folder)